### PR TITLE
refactor external link to component

### PR DESCRIPTION
**Pull request by tianleh**
*Issue #, if available:*

*Description of changes:*
Create a component for external link. 
Also added a few doc links for authentication & authentication.

Lint has optimized some whitespace and wrapping. Verified on UI that the whitespace is still there.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/apps/configuration/utils/display-utils.tsx [hunk 1]
```diff
@@ -23,12 +23,14 @@ import {
...
   EuiButtonProps,
   EuiToolTip,
+  EuiLink,
 } from '@elastic/eui';
 import { isEmpty } from 'lodash';
 
 import React from 'react';
 import { ExpressionModal } from '../panels/expression-modal';
 import { EMPTY_FIELD_VALUE } from '../ui-constants';
+import { DocLinks, LEARN_MORE } from '../constants';
```
Comment (bandinib-amzn) [inline public/apps/configuration/utils/display-utils.tsx:33]: Are we using DocLinks?

File: public/apps/configuration/utils/display-utils.tsx [hunk 2]
```diff
@@ -124,3 +126,11 @@ export function ExternalLinkButton(props: { text: string; href: string } & EuiBu
...
   );
 }
+
+export function ExternalLink(props: { href: string }) {
+  return (
+    <EuiLink external={true} href={props.href} target="_blank">
+      {LEARN_MORE}
+    </EuiLink>
+  );
+}
```
Comment (yuxitang-amzn) [inline public/apps/configuration/utils/display-utils.tsx:136]: Q: Are we able to handle the external icon itself in a separate row?
image

File: public/apps/configuration/utils/display-utils.tsx (hunk 1 above)
Comment (tianleh) [reply public/apps/configuration/utils/display-utils.tsx:33]: Good catch. Removed.

File: public/apps/configuration/utils/display-utils.tsx (hunk 2 above)
Comment (tianleh) [reply public/apps/configuration/utils/display-utils.tsx:136]: It cannot. I just added "display: inline-block" and noticed that it remains as a block during window dragging.

**Review by bandinib-amzn — APPROVED** [review]

**Pull request merged**